internal/services/oracle/oracle_network_anchor_data_source_test.go

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package oracle_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance/check"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/oracle"
+)
+
+type NetworkAnchorDataSource struct{}
+
+func TestNetworkAnchorDataSource_basic(t *testing.T) {
+	data := acceptance.BuildTestData(t, fmt.Sprintf("data.%[1]s", oracle.NetworkAnchorDataSource{}.ResourceType()), "test")
+	r := NetworkAnchorDataSource{}
+	data.DataSourceTest(t, []acceptance.TestStep{
+		{
+			Config: r.basic(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).Key("location").Exists(),
+				check.That(data.ResourceName).Key("name").Exists(),
+				check.That(data.ResourceName).Key("resource_group_name").Exists(),
+				check.That(data.ResourceName).Key("resource_anchor_id").Exists(),
+				check.That(data.ResourceName).Key("subnet_id").Exists(),
```

Comment:
can we add a check for the `id` filed here too?
```suggestion
				check.That(data.ResourceName).Key("subnet_id").Exists(),
				check.That(data.ResourceName).Key("id").Exists(),
```